### DISCUSSION TITLE
Switch to LOC during LAND, when AP is switched OFF

### DISCUSSION THREAD

**Discussion by lve0200** [flybywiresim/aircraft#2361] [Help] [answered]
Hi,
I believe to remember, that when landing and after the AP was switched OFF, the AP switched automatically to LOC to stay on the center line.
I don't see this anymore. Feature or bug ?

**Comment by MapleNAV-TCASFAULT** [comment]
Its supposed to stay on APPR if you're doing an ILS or RNAV approach.

**Reply by MapleNAV-TCASFAULT** [reply]
But I have realized that though APPR mode is activated and the FMA callouts are correct, the flight directors don't work properly unless you reengage the APPR button

**Comment by lve0200** [comment]
Its a bug, though?!

**Reply by MapleNAV-TCASFAULT** [reply]
Hard to understand what's your point. can you rephrase the question?

**Comment by lve0200** [comment]
My basic assumption is that "the systems" should keep the ACFT on center line during landing, when the AP was switched off at DH. I supposed it would switch to LOC, but you corrected me, saying

> Its supposed to stay on APPR

,but then you added

> But I have realized that though APPR mode is activated ... the flight directors don't work properly unless you reengage the APPR button

I assume now that having to reengage APR is not correct and a bug, right?

**Reply by MapleNAV-TCASFAULT** [reply]
Yes, thats right, but its random i believe as some people that I know did not encounter it.

**Reply by MapleNAV-TCASFAULT** [reply]
So to cap it all off; When the AP is disengaged the APPR mode button should still be illuminated, but in some cases it acts as if its not on and will only fly the localizer.

**Reply by lve0200** [reply]
Hhm, there is no "at random" in Software; it may look like, but computers do, what they're supposed to do by programs.
I'll transfer it as a bug, now.

**Comment by lve0200** [comment, marked as answer]
I closed the issue. There's a good reference here:
https://safetyfirst.airbus.com/lateral-runway-excursions-upon-landing/

**Reply by MapleNAV-TCASFAULT** [reply]
I like this reference, but I think this article is more leaned to the “poor airmanship” factors, which I believe is not your case. Unless if I interpreted your question completely wrong.

**Reply by lve0200** [reply]
Its more implicit, since they are not mentioning any automated system activity, I assume its fully manual, as suggested by HighFlyer2010 in closed #2422. After all these explanations, there seems also no additional role for any kind of automation.
